### DISCUSSION TITLE
looking into the future ( >= v1.0.0)

### DISCUSSION THREAD

**Discussion by jacoobes** [sern-handler/handler#108] [General]
Yay! we finally made it to version 1. We patched some bugs, made big revamps, made the website, cli, and much more. Thank you to everyone who's contributed.
Now, looking forward, Here are some features that are planned to come into sern!

# Framework
- [ ] Module Manager
- Interact with the commandModules and eventModules being stored.
- [ ] Scoped Plugins
-  Interact with features of sern and customizing it to its liking
- [ ] Striving to keep performance an importance
- Will look at a few ways to improve overall performance
- [ ] Looking at feature requests 
- [ ] Refactoring Context
- Context class is used to interoperate between message and interaction classes. 
     - Will probably be changed a little in the future, using a better internal data structure
     - Will be able to store data
- I take feature requests seriously, if theres something you'd like, I'll review and decide whether it'll be integrated. (pls feature request)

**Comment by trueharuu** [comment]
hacked by monox

**Reply by jacoobes** [reply]
![joe](https://user-images.githubusercontent.com/76754747/185731107-dd0674f2-ef8b-4135-b24f-20d424dbbfc5.png)
